pipeline/mutate/mutator_hydrator.go

```diff
@@ -172,32 +169,32 @@ func (a *MutatorHydrator) Mutate(r *http.Request, session *authn.AuthenticationS
 	}
 	req.Header.Set(contentTypeHeaderKey, contentTypeJSONHeaderValue)
 
-	var client *http.Client
-
+	client := http.DefaultClient
+	if a.d.Tracer() != nil {
+		client = otelhttp.DefaultClient
+	}
```

Comment:
I'm probably late to the party, but the HTTP default client is quite shitty. It doesn't have timeouts etc configured. Why not keep using

```
 		client:       httpx.NewResilientClient().StandardClient(),
```

as the default?

Reply:
This wasn't the default before this PR. The http.DefaultClient was the fallback in case no retrying was configured.

Let's tackle all the retrying-logic in another PR.